docs/learn/learn-comparison-avalanche.md

```diff
@@ -0,0 +1,104 @@
+
+<!-- TODO: Figure out what a Directed Acyclyc Graph (DAG) is in accordance to a linear blockchain. And state the different in the Polkadot model -->
+
+#### C-chain (Contracts)
+
+The C-chain is where most activity will happen on the Avalanche blockchain. It allows for different virtual machines to execute smart contract code. Out of the box it has support for EVM and AVM(Avalanche VM). 
+
+#### Subnets or sub-networks
+
+Subnets can be public or private and allow for developers to design and launch a blockchain on Avalanche. These are one or many validators that are subsets from the overall validator set of the network. Any validator in the Avalanche network has to validate all three chains making them a part of the primary network subnet. Subnets can validate any blockchains, however each blcokcahin is validated by exactly one subnet. 
+
+<!-- TODO: How different is this model from the parachain model? Do subnets also have to use the avalanche consensus model? They have a choice of VM but do they of consensus? -->
+
+## Consensus 
+
+<!-- TODO: Dig into each Snow protocol in detail -->
+
+Avalanche uses a family of consensus protocols to achieve security, liveness and finality. These are known as the Snow* protocols. This group of protocol composed together use both classical and Nakomoto consensus as well as a Delegated Proof-of-Stake system for its block creators.
+
+The Snow family is a hierarchical collection of systems used to reach finality on Avalanche:
+
+- Slush
+- Snowflake
+- Snowball
+
+
+Compared to Polkadot, Avalanche uses a asynchronous hybrid system that is based on a clasical and Nakomoto approach. Polkadot uses an synchronous hybrid model that combines BABE and GRADNPA, where BABE is the algorithm used to build blocks in a probabilistic way, and GRANDPA is a finality mechanism that uses a deterministic approach to adding blocks to the longest chain. In the end, validators agree to whole chains, rather than single new blocks.
+
+## Staking Mechanics
+
+<!-- TODO: Add Polkadot comparison to become a validator and nominator -->
+
+Avalanche uses a Delegated Proof-of-Stake mechanism without any slashing. The barrier to entry for staking as a full node validator is 2500 AVAX, and 25 AVAX to become a delegator. 
+
+## Shards
+
+<!-- TODO: Add sharding comparison... this will mainly be subnets vs parachains -->
+
+## Message Passing
+
+Avalanche does not have a native trustless message passing mechanism, instead it relies on bridges. Though, because it is an EVM compatible protocol, it's able to interoperate at a token level, however subnets do not have a messaging layer out of the box. Polkadot, with its XCM and XCMP messaging protocols allows for a native and trustless messaging scheme, making composibility of chains to build powerful applications much simpler.
```

Comment:
Rephrasing slightly

```suggestion
Avalanche does not have a native trustless message passing mechanism. Instead, it relies on bridges. Though, because it is an EVM compatible protocol, it's able to interoperate at a token level. However, subnets do not have a messaging layer out of the box. Polkadot, with its XCM and XCMP messaging protocols allows for a native and trustless messaging scheme, thus supporting composability of chains and enabling the development of powerful cross-chain applications.
```